.config/nextest.toml

```diff
@@ -7,3 +7,12 @@ nextest-version = { required = "0.9.55", recommended = "0.9.57" }
 
 [profile.ci]
 fail-fast = false
+
+# TODO(https://github.com/oxidecomputer/omicron/issues/4000): 
+# Remove this override once replicated ClickHouse is the sole modality
+# It is not possible to install both single node and a replicated cluster
+# on the same server. This means that while we are testing both modalities,
+# sometimes the replicated cluster will fail to start due to parallelism.
+[[profile.ci.overrides]]
+filter = 'test(/\btest_build_replicated/)'
+retries = 2
```

Comment:
The [serial_test](https://docs.rs/crate/serial_test/latest) crate does not seem to work with nextest :(

Reply:
Yeah, the model is sadly not compatible. See https://nexte.st/book/test-groups.html for how to do mutual exclusion of tests with nexteat

Reply:
Works thanks! :tada: